ipfsapi/client.py

```diff
@@ -743,6 +743,42 @@ def resolve(self, name, recursive=False, **kwargs):
         args = (name,)
         return self._client.request('/resolve', args, decoder='json', **kwargs)
 
+    def key_gen(self, keyName, type='rsa', size='2048', **kwargs):
```

Comment:
Also please write `key_name` instead of `keyName` – we use snake_case everywhere else, no point in introducing any inconsistency here.